examples/React/src/App.tsx

```diff
@@ -106,6 +106,15 @@ function App() {
                 width='100%'
                 height='700px'
             ></gs-relative-growth-advantage>
+            <gs-mutations-over-time
+                lapisFilter='{ "pangoLineage": "JN.1*", "dateFrom": "2024-01-15", "dateTo": "2024-07-10" }'
```

Comment:
```suggestion
                lapisFilter={JSON.stringify(numerator.lapisFilter)}
```

(and maybe we can change the default pango lineage to `B.1.1.7*` to get more data)